panther_utils/include/panther_utils/common_utilities.hpp

```diff
@@ -72,7 +72,7 @@ std::fstream OpenFile(const std::string & file_path, const std::ios_base::openmo
  * @param version The version to be compared with.
```

Comment:
@delihus The point here was the "Panther" word. Since we made this function more generic, we don't know from this function perspective whether the Panther's software version is verified or sth else ;)